context/context.go

```diff
@@ -363,7 +372,7 @@ func (ctx *FunctionContext) Send(outputName string, data []byte) ([]byte, error)
 
 	payload = data
 
-	if traceable(output.ComponentType) {
+	if isTracingEnabled(ctx) && traceable(output.ComponentType) && !ctx.GetOptions().GetSendWithRawData() {
```

Comment:
What happens if tracing is enabled and the user choose to send with raw data?

Reply:
I see, let me define the priority as follows:

the user decides whether to send raw data > tracing is enabled > target is traceable

Does this make sense?

Reply:
I think the logic is added specially for SkyWalking. Can we replace the `!GetSendWithRawData` with something like `IsTracingProviderSkyWalking`?

If user enables skywalking tracing, it has to be cloud events, else keep the original data.
Does this make sense?

Reply:
done